pkg/v1/tkg/tkgctl/config_cluster.go

```diff
@@ -28,7 +28,7 @@ func (t *tkgctl) ConfigCluster(configClusterOption CreateClusterOptions) error {
 		return err
 	}
 
-	options, err := t.getCreateClusterOptions(configClusterOption.ClusterName, &configClusterOption)
+	options, err := t.getCreateClusterOptions(configClusterOption.ClusterName, &configClusterOption, false)
```

Comment:
why is this always false?

Reply:
If we are not changing for now, put a TODO comment that says “To address since this breaks create --dry-run of a clusterclass-based config”

Reply:
In dry-run use case, existing logic doesn't read ClusterClass input file, it ignores if it's not a legacy config file.
I have created a GitHub issue and added TODO - https://github.com/vmware-tanzu/tanzu-framework/issues/2516